File: 01.JS-Basic/Проверки/toyShop.js
function toyShop(input) {

    let tripPrice = Number(input[0])
    let puzlesCount = Number(input[1])
    let barbyCount = Number(input[2])
    let teddyBear = Number(input[3])
    let minions = Number(input[4])
    let truck = Number(input[5])

    let totalToys = puzlesCount + barbyCount + teddyBear + minions + truck
    let totalIncome = (puzlesCount * 2.6 + barbyCount * 3 + teddyBear * 4.1 + minions * 8.2 + truck * 2) * 0.9

    if (totalToys >= 50) {
        totalIncome = totalIncome * 0.75
    }

    if (totalIncome >= tripPrice) {
        console.log(`Yes! ${(totalIncome - tripPrice).toFixed(2)} lv left.`)
    } else {
        console.log(`Not enough money! ${(tripPrice - totalIncome).toFixed(2)} lv needed.`)
    }



}
toyShop(["320", "8", "2", "5", "5", "1"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatements/ToyShop.java
package ConditionalStatements;

import java.util.Scanner;

public class ToyShop {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        Double holidayPrice = Double.parseDouble(scan.nextLine());
        Double puzzles = Double.parseDouble(scan.nextLine());
        Double barbieToy = Double.parseDouble(scan.nextLine());
        Double teddyBear = Double.parseDouble(scan.nextLine());
        Double minion = Double.parseDouble(scan.nextLine());
        Double truckToy = Double.parseDouble(scan.nextLine());

        Double puzzlesProfit = puzzles * 2.6;
        Double barbieToyProfit = barbieToy * 3;
        Double teddyBearProfit = teddyBear * 4.1;
        Double minionProfit = minion * 8.2;
        Double truckToyProfit = truckToy * 2;

        Double totalProfit = puzzlesProfit + barbieToyProfit + teddyBearProfit + minionProfit + truckToyProfit;

        if (puzzles + barbieToy + teddyBear + minion + truckToy >= 50) {
            totalProfit *= 0.75;
        }
        totalProfit *= 0.9;

        if (totalProfit >= holidayPrice) {
            System.out.printf("Yes! %.2f lv left.", totalProfit - holidayPrice);
        } else {
            System.out.printf("Not enough money! %.2f lv needed.", holidayPrice - totalProfit);
        }
    }
}
